web-app/src/app/screens/Account.tsx

```diff
@@ -589,17 +592,91 @@ export default function APIAccount(): React.ReactElement {
             </div>
             <Typography id='code-block-content'>
               <span style={{ color: '#ff79c6', fontWeight: 'bold' }}>curl</span>{' '}
-              --location &apos;{apiURL}&apos;
-              <span style={{ color: '#f1fa8c' }}>\</span>
+              --location &apos;{apiURL}/metadata&apos;
+              <span style={{ color: theme.mixins.code?.contrastText }}>\</span>
               <br />
-              <span style={{ color: '#f1fa8c' }}>--header</span> &apos;Accept:
-              application/json&apos;
-              <span style={{ color: '#f1fa8c' }}>\</span>
+              <span style={{ color: theme.mixins.code?.contrastText }}>
+                --header
+              </span>{' '}
+              &apos;Accept: application/json&apos;
+              <span style={{ color: theme.mixins.code?.contrastText }}>\</span>
               <br />
-              <span style={{ color: '#f1fa8c' }}>--header</span>
+              <span style={{ color: theme.mixins.code?.contrastText }}>
+                --header
+              </span>
               &apos;Authorization: Bearer [Your Access Token]&apos;
             </Typography>
           </Paper>
+          <Box sx={{ p: 1, mt: 5 }}>
+            <Typography variant='sectionTitle'>
+              How to Get the Access Token in Your App
+            </Typography>
+            <Typography sx={{ mb: 2 }}>
+              Follow{' '}
+              <Link href='mailto:api@mobilitydata.org' color={'inherit'}>
```

Comment:
this link should be pointing to the swagger documentation and open in a new page